Chore/update panel image and key (#703)
centrica-engineering/nucleus-docs: src/content/docs/components/ns-panel.mdx

| Key | Field type | Guidelines |
| :--- | :--- | :--- |

> | <span class="marker">A</span> | Decoration | Choose between a no-colour background (default) or one of the available brand colours. |

Reviewer: Include a link to the list of decoration options in the Specification.

```suggestion
| <span class="marker">A</span> | Decoration | Choose between a no-colour background (default) or one of the [decoration options](#specification-attribute-decoration). |
```

> | <span class="marker">B</span> | Content | Items placed in the anonymous slot can include a variety of different types of content and components. |

Reviewer: ```suggestion
| <span class="marker">B</span> | Anonymous slot | Items placed in the anonymous slot can include a variety of different types of content and components. |
```

> | <span class="marker">C</span> | Columns | Use [`<ns-column>`](/components/ns-column) to define a multi column layout. |

Reviewer: I'd like to suggest removing reference to Columns. As they are added to the anonymous slot as described in [B].

```suggestion
```
